### PR TITLE
chore: create a table for the container flags

### PR DESCRIPTION
**Pull request by MariamFahmy98**
## Related issue #
None

## Proposed Changes
Create a table for the container flags.

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

**Comment by MariamFahmy98** [comment]
A snapshot for the created table:
Screenshot_20231215_151504

**Comment by chipzoller** [comment]
I like this better, especially the default values.

File: content/en/docs/Installation/customization.md [hunk 1]
```diff
@@ -284,75 +284,77 @@ The following flags can be used to control the advanced behavior of the various
...
+| `clientRateLimitBurst` (ABCR) | 300 | Configures the maximum burst for throttling. Uses the client default if zero. |
+| `clientRateLimitQPS` (ABCR) | 300 | Configures the maximum QPS to the API server from Kyverno. Uses the client default if zero. |
+| `disableMetrics` (ABCR) | false | Specifies whether to enable exposing the metrics. |
+| `dumpPayload` (AC) | false | Toggles debug mode. When debug mode is enabled, the full AdmissionReview payload is logged. Additionally, resources of kind Secret are redacted. Should only be used in policy development or troubleshooting scenarios, not left perpetually enabled. |
+| `enableConfigMapCaching` (ABR) | true | Enables the ConfigMap caching feature. |
+| `enableDeferredLoading` (A) | true | Enables deferred (lazy) loading of variables (1.10.1+). Set to `false` to disable deferred loading of variables which was the default behavior in versions < 1.10.0. |
+| `enablePolicyException` (ABR) | true | Set to `true` to enable the [PolicyException capability](/docs/writing-policies/exceptions/). |
+| `enableTracing` (ABCR) | false | Set to enable exposing traces. |
+| `enableTuf` (AR) | Enable tuf for private sigstore deployments. |
```
Comment (chipzoller) [inline content/en/docs/Installation/customization.md:308]: Missing pipe.

**Review by chipzoller — CHANGES_REQUESTED** [review]

**Comment by chipzoller** [comment]
Nice, Mariam.

**Review by chipzoller — APPROVED** [review]

**Pull request merged**